src/Task/Git/CommitMessage.php

```diff
@@ -96,7 +96,18 @@ public function run(ContextInterface $context)
         }
 
         if (count($exceptions)) {
-            return TaskResult::createFailed($this, $context, implode(PHP_EOL, $exceptions));
+            return TaskResult::createFailed(
+                $this,
+                $context,
+                implode(PHP_EOL, $exceptions) . <<<KAWAii
+</>
```

Comment:
Why is this one added? I think it's more clear to just keep the color like the other tasks do.